Library/Formula/kafka.rb

```diff
@@ -14,7 +14,7 @@ class Kafka < Formula
 
   depends_on "gradle"
   depends_on "zookeeper"
-  depends_on :java => "1.7+"
+  depends_on :java => "1.7"
```

Comment:
An easier solution and one I would accept is reverting back 
java => "1.7"  to  java => "1.7+" but that will mean someone who only has Java 1.7 installed will not be able to run the current brewed version of Kafka.  Since the class versions generated by the bottle will be 1.8.